index.html

```diff
@@ -87,27 +77,38 @@
             href: "https://github.com/w3c-dvcg/lds-ecdsa-secp256k1-2019/issues/"
           }]
         }],
+
+        // URI of the patent status for this WG, for Rec-track documents
+        // !!!! IMPORTANT !!!!
+        // This is important for Rec-track documents, do not copy a patent URI from a random
+        // document unless you know what you're doing. If in doubt ask your friendly neighbourhood
+        // Team Contact.
+        wgPatentURI:  "",
+        maxTocLevel: 4,
+        /*preProcess: [ webpayments.preProcess ],
+        alternateFormats: [ {uri: "diff-20111214.html", label: "diff to previous version"} ],
+        */
         localBiblio:  {
-          "RDF-DATASET-NORMALIZATION": {
-            title:    "RDF Dataset Normalization 1.0",
+          "RDF-DATASET-CANONICALIZATION": {
+            title:    "RDF Dataset Canonicalization 1.0",
             href:     "http://json-ld.github.io/normalization/spec/",
             authors:  ["David Longley", "Manu Sporny"],
-            status:   "CG-DRAFT",
+            status:   "CGDRAFT",
             publisher:  "JSON-LD Community Group"
           },
           "SECURITY-VOCABULARY": {
             title:    "Security Linked Data Vocabulary",
             href:     "https://web-payments.org/vocabs/security",
             authors:  ["Manu Sporny","David Longley"],
-            status:   "CG-DRAFT",
+            status:   "CGDRAFT",
             publisher:  "Web Payments Community Group"
           },
           "LD-SIGNATURES": {
             title:    "Linked Data Signatures 1.0",
-            href:     "https://web-payments.org/specs/source/ld-signatures/",
+            href:     "https://w3c-dvcg.github.io/ld-signatures/",
             authors:  ["David Longley", "Manu Sporny"],
-            status:   "CG-DRAFT",
-            publisher:  "Web Payments Community Group"
+            status:   "CGDRAFT",
+            publisher:  "Digital Verification Community Group"
```

Comment:
Seems like my draft is built from a different template, which of these is correct for this WG?

Reply:
I think for "status" actually "Draft Community Group Report" would be most correct. For "publisher" it seems "Digital Verification Community Group" is correct.